.github/testing-issue-template.md

```diff
@@ -32,6 +37,14 @@ Maintainers can promote this to stable by commenting `/promote <rev>[,<rev>] sta
 
 > For example
 >
-> * To promote a single revision, run `/promote <rev> stable`
-> * To promote multiple revisions, run `/promote <rev>,<rev> stable`
-> * To promote a revision and close the issue, run `/promote <rev>,<rev> stable done`
+> - To promote a single revision, run `/promote <rev> stable`
+> - To promote multiple revisions, run `/promote <rev>,<rev> stable`
+> - To promote a revision and close the issue, run `/promote <rev>,<rev> stable done`
+
+You can promote both revisions just built with:
+
+```
+/promote {{ env.revision_amd64 }},{{ env.revision_arm64 }} stable done
+```
+
+cc: @jnsgruk @merlijn-sebrechts
```

Comment:
(Not an issue right now but it's something I'm wondering about for all repo's.)

Is there a reason why you tag us in the issue? I'm subscribed to this repo so I get all issues. My thought would be that people subscribe to the repo's they want to get notified about. That way, we can (eventually) use the same testing template on all repos. What do you think?

Reply:
Yeh happy to remove this.